scene/gui/item_list.cpp

```diff
@@ -395,8 +395,10 @@ bool ItemList::is_same_column_width() const {
 }
 
 void ItemList::set_max_text_lines(int p_lines) {
-
```

Comment:
This is definitely not how to solve the error. If error are triggered that's normal, it means the calling code should check the p_lines value by itself. The the ERR_FAIL_COND is only a barrier avoiding crashes.